fuzz/fuzzers/decode.rs

```diff
@@ -0,0 +1,16 @@
+#![no_main]
+#[macro_use] extern crate libfuzzer_sys;
+extern crate rawloader;
+
+use std::io;
+
+fuzz_target!(|data: &[u8]| {
```

Comment:
The official API is decode_safe() and that will catch panics. If we change this to that will hangs still be caught? Those are of course bugs.